### PR TITLE
Add documentation links for CSS grid and flexbox RSC-481

### PR DESCRIPTION
**Pull request by glenhunter**
https://issues.sonatype.org/browse/RSC-481

### CODE REVIEW

File: gallery/src/pages/StylingComponents.tsx [hunk 1]
```diff
@@ -85,6 +85,48 @@ const StylingRSCPage = () =>
         visual state. These classes are common across all components that might need them, especially in the case of the
         validation utility classes.
       </p>
+
+      <h3 className="nx-h3 nx-tile__section-header">CSS Resources</h3>
```
Comment (rpokorny) [inline gallery/src/pages/StylingComponents.tsx:89]: Shouldn't this be in a separate `.nx-tile__subsection`?

Comment (rpokorny) [inline gallery/src/pages/StylingComponents.tsx:89]: Is this supposed to be `.nx-tile__subsection-header`?

File: gallery/src/pages/StylingComponents.tsx [hunk 2]
```diff
@@ -85,6 +85,48 @@ const StylingRSCPage = () =>
...
+      <p className="nx-p">
+        Many components in the RSC are layed out using CSS Flexbox or CSS Grid. Flexbox is well
```
Comment (rpokorny) [inline gallery/src/pages/StylingComponents.tsx:91]: Wouldn't it be "laid"?

File: gallery/src/pages/StylingComponents.tsx (hunk 1 above)
Comment (glenhunter) [reply gallery/src/pages/StylingComponents.tsx]: > Shouldn't this be in a separate `.nx-tile__subsection`?

Yup: https://github.com/sonatype/sonatype-react-shared-components/pull/308/commits/d92adcbbde5c560e9dbb4d8ce989930c2219fcb6

Comment (glenhunter) [reply gallery/src/pages/StylingComponents.tsx]: > Is this supposed to be `.nx-tile__subsection-header`?

Yup: https://github.com/sonatype/sonatype-react-shared-components/pull/308/commits/d92adcbbde5c560e9dbb4d8ce989930c2219fcb6

File: gallery/src/pages/StylingComponents.tsx (hunk 2 above)
Comment (glenhunter) [reply gallery/src/pages/StylingComponents.tsx]: > Wouldn't it be "laid"?

I was curious so I did a bit of research and while "layed" has become a common alternate spelling "laid" is the correct past tense. https://github.com/sonatype/sonatype-react-shared-components/pull/308/commits/d92adcbbde5c560e9dbb4d8ce989930c2219fcb6

**Review by rpokorny — APPROVED** [review]

**Pull request merged**